cli/flags.rs

```diff
@@ -811,8 +811,22 @@ fn completions_subcommand<'a, 'b>() -> App<'a, 'b> {
     .about("Generate shell completions")
     .long_about(
       "Output shell completion script to standard output.
-  deno completions bash > /usr/local/etc/bash_completion.d/deno.bash
-  source /usr/local/etc/bash_completion.d/deno.bash",
+
+bash:
+  deno completions bash | sudo tee /etc/bash_completion.d/deno.bash > /dev/null
```

Comment:
Why is this preferable to 
```
deno completions bash > /usr/local/etc/bash_completion.d/deno.bash
```
It's significantly more complicated.

Reply:
@ry I think it's the `sudo`. The pipe thing only works if the current shell is invoked with `sudo`, since the shell is doing the writing to the protected file. This lets you put it before `tee`.

Reply:
I don't think it is in the scope of deno to tell users when to use sudo. If you're root this is not a problem.

The shell completion help text is not the place to explain unix file system permissions.

Reply:
The doc wasn't outdated, - on macOS it still works perfectly. It seems it's permission problem on Linux and Windows though.

I've played with deno on windows recently, using bash (Git Bash), and it doesn't have `sudo` at all. I come up with placing bash completions into `~/.completions` and automatically sourcing it on each terminal session with this little code:

https://github.com/denoland/deno/issues/5777#issuecomment-726268305

Not very standard, I haven't seen anyone using this path, but at least it has no permission problems. Whatever path it would be, I think completions should be installed automatically right on installation step (since you have it anyway), so completions would work out of the box!